src/pages/App.tsx

```diff
@@ -132,21 +132,28 @@ function App(): JSX.Element {
                     {!isMobile && <SideBar />}
                     <MainBody
                         ref={bodyRef}
-                        className={`z-0 flex flex-col items-center flex-grow h-full px-4 pt-4 pb-4 overflow-x-hidden overflow-y-auto sm:pt-8 md:pt-10
-                        ${location.pathname === '/goodid' ? 'flex-start' : 'justify-between'}`}
+                        className={`z-0 flex flex-col items-center flex-grow h-full px-4 pt-4 pb-4 overflow-x-hidden overflow-y-auto sm:pt-8
```

Comment:
I think it's time to apply https://www.npmjs.com/package/classnames

```
import classNames from 'classnames'
// ...
const { pathname } = location
const isFV = pathname.startsWith('/goodid')
const isClaim = pathname.startsWith('/claim')
// ..
className={classnames('z-0 flex flex-col items-center flex-grow h-full px-4 pt-4 pb-4 overflow-x-hidden overflow-y-auto sm:pt-8', {
  'flex-start': isFV
  'justify-between': !isFV,
  'xl:pt-0': isClaim,
  'md:pt-10': !isClaim
})}
```

Reply:
added for now.

However I kept my 'todo' comment for later at some point.
I do think here the container of the main page should be defined to be the same for all (which is is and should be!)
whatever styling needs to be applied for specific pages should be done on the page itself, not in the main app

but for now this should be fine